src/rules/property-no-unknown/README.md

```diff
@@ -0,0 +1,119 @@
+# property-no-unknown
+
+Disallow unknown property.
+
+```css
+.foo { heigth: 100%; }
+/**    ↑
+ *  These properties */
+```
+
+This rule considers properties defined in the [CSS Specifications and browser specific properties](https://github.com/betit/known-css-properties#source) to be known.
+
+## Options
+
+### `true`
+
+The following patterns are considered warnings:
+
+```css
+.foo {
+  colr: blue;
+}
+```
+
+```css
+.foo {
+  -moz-align-self: center;
+}
+```
+
+The following patterns are *not* considered warnings:
+
+```css
+.foo {
+  color: green;
+}
+```
+
+```css
+.foo {
+  fill: black;
+}
+```
+
+```css
+.foo {
+  -webkit-align-self: center;
+}
+```
+
+```css
+.foo {
+  align-self: center;
+}
+```
+
+```css
```

Comment:
Can you remove this and the next for examples as they can covered with the addition of the "This rule ignores variables (`$sass`, `@less`, `--custom-property`)." sentence.